Gems/LyShine/Code/Source/UiTextComponent.cpp

```diff
@@ -3381,6 +3381,15 @@ void UiTextComponent::Activate()
     FontNotificationBus::Handler::BusConnect();
     LanguageChangeNotificationBus::Handler::BusConnect();
 
+#if defined(CARBONATED)
+    // Reload the font families if they were deleted while this UiTextComponent was inactive.
+    // It is important for UiTextComponent which can be activated and deactivated many times.
+    if (m_fontFamily->familyName.empty())
```

Comment:
You can check if all these are nullptrs, then there is no need in emptying the name in AtomFont.cpp
    IFFont* normal;
    IFFont* bold;
    IFFont* italic;
    IFFont* boldItalic;

Reply:
Updated